skins/oasis/modules/templates/PageHeader_Index.php

```diff
@@ -7,12 +7,20 @@
 	}
 }
 
+if ( $showCuratedContentToolBtn ) {
+	$curatedContentToolButton = '<a id="CuratedContentTool" class="wikia-button">' . wfMsg( 'wikiacuratedcontent-edit-mobile-main-page' ) . '</a>';
```

Comment:
This whole file seems wrong and is a candidate for refactoring. CONCF-989 was estimated at 3 story points. Do you think we can do the refactor in that time?